lib/scarpe/widget.rb

```diff
@@ -69,6 +75,7 @@ def value_text=(new_text)
     end
 
     def remove_self
+      @@document_root.children.delete(self)
```

Comment:
Need to do this or the element just immediately will get re-rendered. But I now realize that this assumes everyone is a child of document root, but that probably will not remain true.